core/jvm-native/src/main/scala/zio/internal/ZScheduler.scala

```diff
@@ -317,7 +317,10 @@ private final class ZScheduler(autoBlocking: Boolean) extends Executor {
                       if (nRunnables > 0) {
                         val iter = runnables.iterator
                         runnable = iter.next()
-                        if (nRunnables > 1) localQueue.offerAll(iter, nRunnables - 1)
+                        if (nRunnables > 1) {
+                          val iterable = iter.toIterable
+                          localQueue.offerAll(iterable)
+                        }
```

Comment:
Why is this change needed?

Reply:
Yes, we need a change inorder to support for jvm-native a minor change

Reply:
I don't understand this. Can you please elaborate? This will make the JVM runtime slower, which we cannot tolerate. The performance of the JVM runtime should not be affected in any way for another runtime